.pipelines/cherrypick/CherryPick.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Cherry Pick commits from PR
+
+trigger:
+- fasttrack/*
+- tvuong/fasttrack/*
+pr: none
+
+variables:
+  - name: pipelinesRepoFolder
+    value: 'CBL-Mariner-Pipelines'
+
+  - name: pipelinesRepoRoot
+    value: $(Agent.BuildDirectory)/$(pipelinesRepoFolder)
+
+resources:
+  repositories:
+    - repository: CBL-Mariner-Pipelines
+      type: git
+      name: mariner/CBL-Mariner-Pipelines
+
+jobs:
+  - job: 'CherryPick'
+    pool: mariner-core-x64-1es-ubuntu-22.04
+    timeoutInMinutes: 15
+    displayName: 'Cherry-pick commit'
+
+    steps:
+    - checkout: CBL-Mariner-Pipelines
+      path: $(pipelinesRepoFolder)
+      displayName: 'Checkout CBL-Mariner'
```

Comment:
Can we move `chekout` inside `CherryPick.yml`? This way it'll have control over the folder where the repository is cloned and we can drop the `variables` here and give more flexibility to the template.

Reply:
sorry, I'm a bit confused, do you mean moving the checkout step to `CherryPick.yml` of CBL-Mariner-Pipelines?

Reply:
Ah, both files have the same name - I didn't see that. To answer your question: both. I meant moving the `checkout` step to `CherryPick.yml` in `CBL-Mariner-Pipelines`.:)